apps/explorer-e2e/src/integration/node-switcher.cy.js

```diff
@@ -71,7 +71,7 @@ context('Node switcher', function () {
     });
 
     afterEach('Close node switcher', function () {
-      cy.getByTestId(closeDialogBtn).click();
+      cy.getByTestId(closeDialogBtn).should('be.be.visible').click();
```

Comment:
question (non-blocking): this kinda looks like a typo, but i'm not sure if `be.be.(...)` is any different from just stating `be.(...)`?

Reply:
You're right. I've got no idea why nothing has complained and how the tests passed. But it's corrected!